lib/Dialects/LinalgTransform/Transforms/TransformInterpreter.cpp

```diff
@@ -576,13 +577,45 @@ static LogicalResult executeSequence(linalg::transform::SequenceOp sequence,
   // Stash the function for the cases where we need to apply function-wise
   // transforms.
   FuncOp func = target->getParentOfType<FuncOp>();
+  ModuleOp module = func->getParentOfType<ModuleOp>();
+
+  MLIRContext *ctx = target->getContext();
+  RewritePatternSet patternList(ctx);
+  for (Dialect *dialect : ctx->getLoadedDialects())
+    dialect->getCanonicalizationPatterns(patternList);
+  for (RegisteredOperationName op : ctx->getRegisteredOperations())
+    op.getCanonicalizationPatterns(patternList, ctx);
+  FrozenRewritePatternSet patterns(std::move(patternList));
```

Comment:
There are some tiling enabling patterns that I am not 100% sure are canonicalizations.
@Mogball's PatternSets could help organize things.
No change needed here.

Reply:
Then they should not run here, which happens after every transformation, but within the tiling transformation itself IMO.